packages/contracts/test/exchange/libs.ts

```diff
@@ -77,96 +77,4 @@ describe('Exchange libs', () => {
             });
         });
     });
-
-    describe('LibMath', () => {
-        describe('isRoundingError', () => {
```

Comment:
I *think* the the new test cases in the combinatorial tests for `isRounding` error are sufficient. However, let me know if I've missed something and we want to keep some of these tests around.

Reply:
After looking at this in more depth I decided to re-add some test cases here for `isRoundingError` that are not covered by the combinatorial tests.